projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -910,6 +1431,41 @@ export function registerGetQuoteRoute(shopName: string) {
   }).as(GET_QUOTE_ALIAS.substring(1)); // strip the '@'
 }
 
+/**
+ * Registers POST quote route.
+ */
+export function registerPostQuoteRoute() {
+  log('Registers POST quote route.', registerPostQuoteRoute.name);
+  cy.intercept({
+    method: 'POST',
+    path: `*`, //ToDo: specify the path
+  }).as(POST_QUOTE_ALIAS.substring(1)); // strip the '@'
+}
+
+/**
+ * Registers PATCH quote route.
+ */
+export function registerPatchQuoteRoute() {
+  log('Registers PATCH quote route.', registerPatchQuoteRoute.name);
+  cy.intercept({
+    method: 'PATCH',
+    path: `*`, //ToDo: specify the path
```

Comment:
Here is the same:
Please, specify the path, otherwise when more than one `PATCH` comes then it is not clear which one is meant.

Reply:
done